src/helpers/networkInfo.ts

```diff
@@ -61,6 +61,19 @@ const networkInfo = {
     gasToken: 'MATIC',
     value: networkTypes.polygon,
   },
+  [`${networkTypes.bsc}`]: {
+    balance_checker_contract_address:
+      '0x400A9f1Bb1Db80643C33710C2232A0D74EF5CFf1',
+    color: '#F0B90B',
+    disabled: true,
+    exchange_enabled: false,
```

Comment:
this is only applicable if disabled is false, but this will block it from showing up in swap